source/_integrations/simplisafe.markdown

```diff
@@ -153,6 +153,29 @@ triggers:
         last_event_type: doorbell_detected
 ```
 
+### Using Secret Alerts for Sensor Changes
```

Comment:
```suggestion
### Using secret alerts for sensor changes
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization) also in headings